sap-ha-support/xml/art-sap-ha-support.xml

```diff
@@ -0,0 +1,829 @@
+  </informalfigure>
+  <para>
+   The support details are for high level overview only. Refer to the official documentation
+   for the full conditions.
+  </para>
+  <table xml:id="table-sap-ha-solutions-s4hana">
+   <title>Supported configurations for &s4hana; based on ABAP Platform 1809 or newer</title>
+   <tgroup cols="3">
+    <thead>
+     <row>
+      <entry>Supported configuration</entry>
+      <entry>Status</entry>
+      <entry>Support details</entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>2-node cluster </entry>
+      <entry>Supported <link xlink:href="&dsc;/sbp/all/html/SAP-S4HA10-setupguide-sle15/index.html">Documentation</link></entry>
+      <entry>In a 2-node cluster, ASCS fails over to the same node where ERS is running.</entry>
+     </row>
+     <row>
+      <entry>3-node cluster </entry>
+      <entry>Supported <link xlink:href="&dsc;/sbp/sap/html/SAP-S4HA10-setupguide-simplemount-sle15/index.html">Documentation</link></entry>
+      <entry>In a 3-node cluster, ASCS fails over to the online node where ERS is not running.</entry>
+     </row>
+     <row>
+      <entry>Simple mount file system structure </entry>
+      <entry>Supported <link xlink:href="&dsc;/sbp/sap/html/SAP-S4HA10-setupguide-simplemount-sle15/index.html">Documentation</link></entry>
+      <entry>Shared filesystem mounts are NOT managed by the cluster. This is the recommended
+      configuration over the 'Filesystem' resource-based solution.</entry>
+     </row>
+     <row>
+      <entry>'Filesystem' resource-based </entry>
+      <entry>Supported <link xlink:href="&dsc;/sbp/all/html/SAP-S4HA10-setupguide-sle15/index.html">Documentation</link></entry>
+      <entry>Shared filesystem mounts are managed by the cluster via the 'Filesystem' resource agent.
+      &suse; still supports this configuration, but it is not recommended for new installations.
+      Use the simple mount file system structure instead.</entry>
+     </row>
+     <row>
+      <entry>Multi-SID </entry>
+      <entry>Supported <link xlink:href="&dsc;/sbp/all/html/SBP-SAP-MULTI-SID/index.html">Documentation</link></entry>
+      <entry>Multiple &sap; ASCS/SCS clustered instances are supported in the same cluster.</entry>
+     </row>
+     <row>
+      <entry>Additional dialog or other instances in cluster </entry>
+      <entry>Supported but undocumented</entry>
+      <entry>Although it is possible to run Application Servers in the same cluster where ASCS/ERS
+      are running, it is not recommended for easy management of the cluster.</entry>
+     </row>
+    </tbody>
+   </tgroup>
+  </table>
+ </sect1>
+
+ <sect1 xml:id="sap-ha-solutions-netweaver">
+  <title>HA solutions for &netweaver; based on ABAP Platform 1709 or older</title>
+  <para>
+   Under the &ensa1long; (&ensa1;), the ASCS has to failover to the cluster node where the active
+   ERS is running, because it needs to access the shared memory that stores the enqueue replication table.
```

Comment:
```suggestion
   Under the &ensa1long; (&ensa1;), the ASCS has to fail over to the
   cluster node where the active ERS is running, because it needs to
   access the shared memory that stores the enqueue replication table.
```